coalib/results/Diff.py

```diff
@@ -354,6 +354,16 @@ def delete_line(self, line_nr):
         linediff.delete = True
         self._changes[line_nr] = linediff
 
+    def delete_lines(self, line_nr_start, line_nr_end):
+        """
+        Delete lines in a range, exclusive.
+        """
+        for line_nr in range(line_nr_start, line_nr_end):
+            # Flatten delete_lines to reduce computations
```

Comment:
so you're essentially trying to optimize away a function call at the expense of code duplication? This kind of thing usually only gives a very very limited performance improvement and increases the risk of having bugs/decreases maintainability.